projects/1.20/assets/create-the-kitchen-must-grow/kitchen_grow/lang/zh_cn.json

```
@@ -0,0 +1,110 @@
+{
+  "block.kitchen_grow.beer": "啤酒",
+  "block.kitchen_grow.berry_ice_cream_block": "浆果冰淇淋块",
+  "block.kitchen_grow.chocolate_ice_cream_block": "巧克力冰淇淋块",
+  "block.kitchen_grow.compressor": "压缩机",
+  "block.kitchen_grow.distiller_output": "蒸馏器",
+  "block.kitchen_grow.empty_snow_golem_freezer": "空的雪傀儡制冰室",
+  "block.kitchen_grow.fermentation_barrel": "发酵桶",
+  "block.kitchen_grow.ice_cream_block": "冰淇淋块",
+  "block.kitchen_grow.lemon_ice_cream_block": "柠檬冰淇淋块",
+  "block.kitchen_grow.polished_acacia_planks": "打磨的金合欢木板",
+  "block.kitchen_grow.polished_birch_planks": "打磨的白桦木板",
+  "block.kitchen_grow.polished_crimson_planks": "打磨的绯红木板",
+  "block.kitchen_grow.polished_dark_oak_planks": "打磨的深色橡木木板",
+  "block.kitchen_grow.polished_jungle_planks": "打磨的丛林木板",
+  "block.kitchen_grow.polished_mangrove_planks": "打磨的红树木板",
+  "block.kitchen_grow.polished_oak_planks": "打磨的橡木木板",
+  "block.kitchen_grow.polished_spruce_planks": "打磨的云杉木板",
+  "block.kitchen_grow.polished_warped_planks": "打磨的诡异木板",
+  "block.kitchen_grow.snow_golem_freezer": "雪傀儡制冰室",
+  "block.kitchen_grow.soda": "苏打",
+  "block.kitchen_grow.sour_milk": "酸奶",
+  "block.kitchen_grow.zinc_fluid_tank": "锌流体储罐",
+  "create.goggles.distiller.info": "蒸馏器:",
```

```suggestion
  "create.goggles.distiller.info": "蒸馏器：",
```
【要求】标点符号应使用规范的汉语全角标点符号，具体要求请阅读我们的 [翻译规范](https://cfpa.site/TransRules/) 中 **03.标点符号** 这一节。
诸如这类的问题请在您翻译的所有文件中全部查明并修改。

---

已修改，谢谢指正